src/components/generic/Modal/Modal.scss

```diff
@@ -1,5 +1,34 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-modal {
-  .md-modal__content {
-    padding: 2rem;
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .modal-content {
+    width: 500px;
```

Comment:
the size of the component should come from outside, from the optional className prop

for this component, the optional class should be put on the .modal-content div, not on the big container.